docs/setup/activate-conda-env.md

```diff
@@ -0,0 +1,98 @@
+# Step 6: Activate the Conda Environment for `assign0`
+
+[Getting Started as a Student](./getting-started.md)
+
+[previous: Conda Installation](./conda-install.md)
+
+[next: Build the Zero-To-AI Documentation](./build-z2ai-documentation.md)
+
+Here is a fantastic
+[cheat sheet](https://docs.conda.io/projects/conda/en/4.6.0/_downloads/52a95608c49671267e40c689e0bc00ca/conda-cheatsheet.pdf)
+for working with conda environments. Bookmark it!
+
+## Conda Environments
+
+A conda "environment" is provides virtualization around dependencies. Instead
+of installing a bunch of dependencies directly on your computer that will
+always be accessed any time you try to run Python, conda proposes the notion
+that each environment can have its own custom tailored version of dependencies.
+
+As an example, if I'm working on a python project called `personal-website`,
+perhaps I have a conda environment called `website` for that project which I
+have set up. I can issue a
+```bash
+conda activate website
+```
+which will activate the `website` conda environment, where all my dependencies
+for that project will be stored (including Python version for that specific
+project). I can view the dependencies for that project
+using a
+```bash
+conda list
+```
+which shows all the dependencies and their corresponding versions. If I now
+want to work on a different python project on my same machine that has a
```

Comment:
Check for consistent capitalization of python--I think you usually have it capitalized, but sometimes it's not.